Core 573

sri2k1us opened the pull request:
- This is on top of Apps Listing PR. So reviewing last [commit](https://github.com/cyverse-de/ui/pull/455/commits/1b54fd8a3d4e3942c4dd6d5eab4a1f8b80502300) is enough.

![image](https://user-images.githubusercontent.com/1250790/66219983-2b75c000-e69a-11e9-8ae4-8a40eb9b382f.png)

ash3rz commented on react-components/package.json:
```diff
@@ -20,7 +20,7 @@
     },
     "dependencies": {
-        "@cyverse-de/ui-lib": "^0.3.8",
+        "@cyverse-de/ui-lib": "file:../../ui-lib/cyverse-de-ui-lib-0.3.8.tgz",
```
Reminder to change before merging 😉

ash3rz commented on react-components/src/apps/details/AppInfoDialog.js:
```diff
@@ -33,6 +33,13 @@ import Tab from "@material-ui/core/Tab";
+    APP_DETAIL_TAB: 0,
+    QUICK_LAUNCH_TAB: 1,
+    TOOL_DETAILS_TAB: 2,
+    APP_DOC: 3,
```
👍

sri2k1us commented:
> LGTM 🚢
> Unrelated to this PR though, is the App dot menu the same as the Analysis dot menu? The padding/spacing between the menu items seems more condensed

Good Catch. Removed the padding that i had added and that fixed it.

psarando commented on react-components/src/apps/listing/AppGridListing.js:
```diff
@@ -243,8 +251,9 @@ function getTableColumns(deletable, enableMenu) {
             name: AppFields.RATING.fieldName,
             enableSorting: true,
-            key: AppFields.RATING.key,
+            key: "rating",
```
Another rebase mishap?

sri2k1us replied:
Another one 🤦‍♂

sri2k1us commented on de-lib/src/main/java/org/iplantc/de/apps/client/views/details/AppDetailsViewImpl.java:
```diff
@@ -26,6 +26,8 @@
     private final App app;
     private final AppDetailsAppearance appearance;
     private String searchText;
+    private String searchRegexPattern;
```
Oops...I made a Boo Boo 😖

psarando replied:
Should this line be removed now?

sri2k1us replied:
Yes, that line can be removed. Not sure why the rebase didn't catch that.

sri2k1us commented:
Thanks for the reviews. And @psarando 🦅 👀
